swagger_parser/test/parser/data_classes_test.dart

```diff
@@ -487,6 +487,7 @@ void main() {
               description: 'data',
               jsonKey: 'data',
               isRequired: false,
+              mapType: 'string',
```

Comment:
Write a separate test for this if it's not difficult

Reply:
I added my json sample and wrote a test for swagger v2 additional properties